packages/bits/src/lib/wizard-v2/types.ts

```diff
@@ -2,3 +2,21 @@
 export interface IWizardState {
     finished: boolean;
 }
+
+export interface IWizardConfig {
+    stepStateIcons: IWizardStepStateIconConfig;
+    [key: string]: any;
+}
+
+export interface IWizardStepStateIconConfig {
+    icons?: Partial<IWizardIconStates>;
+    colors?: Partial<IWizardIconStates>;
+}
+
+export interface IWizardIconStates {
+    active: string;
+    visited: string;
+    initial: string;
+    error: string;
+    [key: string]: string;
```

Comment:
maybe the type of the members could be genericized with `<T>` to allow a mapping of states to any type in the future?

Reply:
The angular cdk explicitly defines the type and enum for the step states as string values. I can't think of any other type anybody would want to act there as a state name/value.

![image](https://user-images.githubusercontent.com/22597043/121341357-409ff900-c929-11eb-9c7d-d6d4abe6da36.png)

We can always generalize it later when we ever have the demand for it, can't we?

Reply:
Well, we're using the state map to map a state to a configuration item in our current case. The configuration items we're currently dealing with happen to be strings, but they could just as well be numbers, booleans, or even objects.